lookout/style/format/features.py

```diff
@@ -199,21 +196,17 @@ def __init__(self, language: str, *, siblings_window: int = 5, parents_depth: in
             # It's normal for some languages not to have a token_unwrappers module.
             self.token_unwrappers = {}
 
-        # Order is important and should be consistent with _inplace_write_vnode_features function
-        # where features generation happens.
+        def prefix(prefix, l):
```

Comment:
prefix_sequence_elements

Also the types are missing

Reply:
it is outdated. we do not need it anymore